SPECS/fontconfig/fontconfig.spec

```diff
@@ -68,20 +68,31 @@ make -k check
 %{_mandir}/man3/*
 
 %changelog
+* Wed Feb 02 2022 Cameron Baird <cameronbaird@microsoft.com> - 2.13.95-1
```

Comment:
It won't let me comment on the offending line, but could you remove the `Provides: pkgconfig(fontconfig)` statement in the main package? It's both in the wrong place and not necessary.